### PR TITLE
[FCL-366] Fix for misaligned focus

### PR DESCRIPTION
**Pull request by Donna-H**
## Changes in this PR:
fix hover on the cards at bottom of static pages
## Jira card / Rollbar error (etc)
https://national-archives.atlassian.net/jira/software/projects/FCL/boards/136?selectedIssue=FCL-366
### 
before
Before

### After
after

- [ ] Requires env variable(s) to be updated

### CODE REVIEW

File: ds_judgements_public_ui/sass/includes/govuk_overrides/_grid.scss [hunk 1]
```diff
@@ -155,9 +155,7 @@
   max-width: 23rem;
   height: 16rem;
   padding: $space-1 $space-5;
-  margin-top: -15px;
-  margin-top: 20px;
-  margin-left: -9px;
+  margin: 10px;
```
Comment (jlhdxw) [inline ds_judgements_public_ui/sass/includes/govuk_overrides/_grid.scss:158]: Can we use a spacing value here to ensure our spacing is consistent? I think $space-3 should be about equivalent here. Same comment to the other 10px below

File: ds_judgements_public_ui/sass/includes/govuk_overrides/_grid.scss [hunk 2]
```diff
@@ -140,10 +140,10 @@
 .govuk-grid-column-two-thirds__info_terms_use {
```
Comment (jacksonj04) [inline ds_judgements_public_ui/sass/includes/govuk_overrides/_grid.scss:140]: Just with an eye to simplifying and reducing duplication; is there a reason these classes are so specific?

Comment (Donna-H) [reply ds_judgements_public_ui/sass/includes/govuk_overrides/_grid.scss:140]: @jacksonj04 yes it was based on design checks with Terry to make the cards align correctly as text amopunt etc and titles are different

File: ds_judgements_public_ui/sass/includes/govuk_overrides/_grid.scss (hunk 1 above)
Comment (Donna-H) [reply ds_judgements_public_ui/sass/includes/govuk_overrides/_grid.scss:158]: @jlhdxw updated to $space-2 as per design review with Terry it ligns up best to 10px thank you

File: ds_judgements_public_ui/sass/includes/govuk_overrides/_grid.scss (hunk 2 above)
Comment (jacksonj04) [reply ds_judgements_public_ui/sass/includes/govuk_overrides/_grid.scss:140]: If we're doing something where each box needs different styles with some straightfoward text then there's probably something else we should be looking at, but that's a separate issue! 😄

**Review by jlhdxw — APPROVED** [review]

**Pull request merged**